src/platforms/android/configuration/app-not-respond.mdx

```diff
@@ -4,6 +4,21 @@ sidebar_order: 3
 description: "Learn how to turn off or specify ANR."
 ---
 
+The Android SDK uses two implementations to detect ANRs:
+
+- Reading [ApplicationExitInfo](https://developer.android.com/reference/android/app/ApplicationExitInfo) for devices running Android 11 and above (v2)
+- A Watchdog thread for devices running Android version below 11 (v1)
+
+The new implementation (v2) is more accurate and enables us to be on-par with Google Play Console with regards to reported ANR events and [ANR rate](/platforms/android/performance/instrumentation/performance-metrics/#application-not-responding-anr-rate). We also can capture a thread dump with some additional information like held locks to help you resolve ANRs more efficiently.
```

Comment:
`m`: 
It may make sense not to mention it's more accurate but only mention that it's using the same data source as Google Play Console. As you point out, the downside is that you get less context with v2.

Reply:
sounds good, I'll change the wording